lib/ui/process_ui/widgets/custom_checkbox.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:flutter/src/widgets/placeholder.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:provider/provider.dart';
+import 'package:registration_client/model/field.dart';
+import 'package:registration_client/provider/global_provider.dart';
+import 'package:registration_client/utils/app_config.dart';
+
+class CustomCheckbox extends StatelessWidget {
```

Comment:
can we be consistent in naming the control types? and some control types are under demographic_details package and some are under widgets.
lets create a package control_type
textbox_control
button_control
dropdown_control
html_control
age_date_control
file_upload_control
checkbox_control
biometrics_scan_control